fastify.js

```diff
@@ -408,9 +419,15 @@ function build (options) {
     }
 
     validateBodyLimitOption(opts.jsonBodyLimit)
-    var jsonBodyLimit = opts.jsonBodyLimit || _fastify._jsonBodyLimit
 
-    _fastify.after((notHandledErr, done) => {
+    const onRoute = fastIterator(_fastify._globalHooks.onRoute, _fastify)
```

Comment:
using `fastIterator`  this way completely remove any performance benefits we might have.

Reply:
I know but it's not a hot path and in this way, I can reuse the iterator functionality.

Reply:
Do you prefer a simple loop?